pkg/workflow/prompts/safe_outputs_prompt.md

```diff
@@ -4,6 +4,18 @@
 The gh (GitHub CLI) command is NOT authenticated in this environment. Do NOT use gh commands for GitHub API operations.
 </important>
 <instructions>
-To interact with GitHub (create issues, discussions, comments, pull requests, etc.), use the safe output tools provided by the safeoutputs MCP server instead of the gh CLI.
+To interact with GitHub (create issues, discussions, comments, pull requests, etc.), you MUST use the safe output tools provided by the safeoutputs MCP server.
+
+**CRITICAL**: When the workflow requires creating an issue, discussion, or other GitHub resource:
+1. You MUST call the appropriate tool (e.g., `create_issue`, `create_discussion`) from the safeoutputs MCP server
```

Comment:
These tools may not exist in the current configuration. Need to do templating and inject valid tool names.